content/en/docs/aws/authentication.md

```diff
@@ -65,7 +65,7 @@ plugins:
       ....
```

Comment:
_Further suggestions_

Line 50: second person "you" is preferred (instead of first person "we")

```diff
- Once a user pool created, we will have a `UserPoolId`, a Cognito Domain name, and a Cognito Pool Arn.		
+ Once a user pool created, your will have a `UserPoolId`, a Cognito Domain name, and a Cognito Pool Arn.
```

Line 52:

```diff
- Before you `kfctl apply -V -f ${CONFIG_FILE}`, please update Cognito spec in your Kubeflow
+ Before you run the `kfctl apply -V -f ${CONFIG_FILE}` command, you should update the Cognito specifications in your Kubeflow
```

cc @Jeffwan @PatrickXYS